services/service-runner/app/service/service-runner.ts

```diff
@@ -320,6 +320,8 @@ export async function runServiceFromPull(
 
     let retryCount = 0;
     const maxRetries = 5;
+    let retryDelay = 3_000; // time in ms to wait before retrying
```

Comment:
I think I want to see a slightly longer delay - maybe 5 seconds which would result in a little over 2 and a half minutes total to exhaust all retries.